bin/main.rb

```diff
@@ -1,76 +1,97 @@
 #!/usr/bin/env ruby
+require_relative '../lib/game.rb'
+
+WINNING_COMBINATION = [
+  [0, 1, 2], # top row
+  [3, 4, 5], # middle row
+  [6, 7, 8], # bottom row
+  [0, 4, 8], # left to right diagonal
+  [2, 4, 6], # right to left diagonal
+  [0, 3, 6], # left-most column
+  [1, 4, 7], # middle column
+  [2, 5, 8] # right-most column
+].freeze
+
+board = %w[1 2 3 4 5 6 7 8 9]
+game_tokens = %w[X O]
```

Comment:
Move it also. Because it is here to only be sent to `Game` class method in parameter